test/org/zaproxy/zap/utils/LocaleUtilsUnitTest.java

```diff
@@ -160,4 +163,28 @@ public void shouldNotMatchInvalidMessagesPropertiesFilesWithCreateMessagesProper
             assertThat(file, pattern.matcher(file).matches(), is(equalTo(false)));
         }
     }
+
+    @Test
+    public void shoudAvailableLocalesBeNonEmpty() {
+        // Given
+        List<String> locales = LocaleUtils.getAvailableLocales();
+
+        // When
+        int localesSize = locales.size();
```

Comment:
This statement could be removed (and the "When" word moved to same comment as "Given").